src/control/server/server.go

```diff
@@ -243,6 +244,7 @@ func (srv *server) createServices(ctx context.Context) (err error) {
 		control.WithClientComponent(build.ComponentServer),
 		control.WithConfig(cliCfg),
 		control.WithClientLogger(srv.log))
+	srv.client = rpcClient
```

Comment:
To avoid confusion I might call the variable controlClient or rpcClient instead of just "client" since we have a number of server/client communication relationships happening among various services on an engine.

Reply:
Renamed.